src/Configuration.cpp

```diff
@@ -120,25 +122,43 @@ void Configuration::Load()
     // FTL_SERVICE_METADATAREPORTINTERVALMS -> ServiceConnectionMetadataReportInterval
     if (char* varVal = std::getenv("FTL_SERVICE_METADATAREPORTINTERVALMS"))
     {
-        serviceConnectionMetadataReportInterval = std::chrono::milliseconds(std::stoi(varVal));
+        serviceConnectionMetadataReportInterval = std::chrono::milliseconds(std::stoul(varVal));
     }
 
     // FTL_MAX_ALLOWED_BITS_PER_SECOND -> MaxAllowedBitsPerSecond
     if (char* varVal = std::getenv("FTL_MAX_ALLOWED_BITS_PER_SECOND"))
     {
-        maxAllowedBitsPerSecond = std::stoi(varVal);
+        maxAllowedBitsPerSecond = std::stoul(varVal);
     }
 
     // FTL_ROLLING_SIZE_AVG_MS -> RollingSizeAvgMs
     if (char* varVal = std::getenv("FTL_ROLLING_SIZE_AVG_MS"))
     {
-        rollingSizeAvgMs = std::stoi(varVal);
+        rollingSizeAvgMs = std::stoul(varVal);
     }
 
     // FTL_NACK_LOST_PACKETS -> IsNackLostPacketsEnabled
     if (char* varVal = std::getenv("FTL_NACK_LOST_PACKETS"))
     {
-        nackLostPackets = std::stoi(varVal);
+        nackLostPackets = std::stoul(varVal);
+    }
+
+    // FTL_PLAYOUT_DELAY_MIN_MS and FTL_PLAYOUT_DELAY_MAX_MS -> PlayoutDelay
+    {
+        char* minVal = std::getenv("FTL_PLAYOUT_DELAY_MIN_MS");
+        char* maxVal = std::getenv("FTL_PLAYOUT_DELAY_MAX_MS");
+
+        if (minVal || maxVal) {
+            if (!PLAYOUT_DELAY_SUPPORT) {
```

Comment:
Braces on new line pls [:>](https://theprofoundprogrammer.com/post/26955601996/text-braces-go-on-the-next-line-you-tremendous)

Reply:
![image](https://user-images.githubusercontent.com/52513/146311897-11153870-8dcb-4207-8ef8-653dda10b4cd.png)

thx for the catch :)